roles/storage_backup/README.md

```
@@ -0,0 +1,45 @@
+storage_backup_dir: /opt/backup
+
+storage_backup_script_path: /opt/backup.sh
+storage_backup_compression: zstd
+storage_backup_retention: 30
+storage_backup_minute: 30
+storage_backup_hour: 4
+```
+
+### Variables that have to be configured:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)